tests/page-objects/reports/reportsPage.wdio.js

```diff
@@ -0,0 +1,18 @@
+const reportListID = '#reports-list';
+const reportBodyDetails = '#reports-content .report-body .details';
+const caseId = () => $(`${reportBodyDetails} > ul > li > p > span > a`);
+const caseIdLabel = () => $(`${reportBodyDetails} ul > li > label > span`);
+const submitterPlace = () => $('.position a');
+const submitterPhone = () => $('.sender .phone');
+const submitterName = () => $('.sender .name');
+const  firstReport = () => $(`${reportListID} li:first-child`);
+
+module.exports = {
+  firstReport,
+  submitterName,
```

Comment:
Copying @newtewt comment for reference:

```
Personally I think we should export the element and then interact(click, addValue, getText,etc..) in the spec. It's a little confusing with the mix.
```

I have a different opinion here. I believe page object files should hide the implementation details about which DOM interaction technology they are using (in this case wdio) and test case files (.spec) should just interact with page's behaviors like `getCaseLabel(...)` or `submitCase()`. This will make test cases agnostic of underlying DOM interaction library we are using and possibly in future we can use same test cases with different page object later on if we need to move on from wdio. Thoughts?

Reply:
Good point. Opinions differ on some of these things and best practices, but keeping the `.spec` file a lot less framework dependent helps both readability and maintainability, especially when tools do change. We can have a conversation in a QA sync for consistency.